lib/rake/circle.rake

```diff
@@ -4,6 +4,10 @@ require 'cdo/git_utils'
 RUN_UI_TESTS_TAG = '[test ui]'
 RUN_ALL_TESTS_TAG = '[test all]'
 
+def random_string_of_length(n)
+  rand(36**n).to_s(36)
+end
```

Comment:
Good call. Wasn't sure if the `-i` argument had requirements around non-alphanumeric key elements, but testing now looks like it handles SecureRandom.uuid's `-`s no problem.